<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fereastra Principala</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="ORL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ORL</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">chooseDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="Essex">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Essex</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">chooseDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="CTOVF">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CTOVF</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">chooseDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="dbc60">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>60% training, 40% testing</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">configureDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="dbc80">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>80% training, 20% testing</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">configureDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="dbc90">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90% training, 10% testing</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">configureDB</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="nn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NN</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">algoritm</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="KNN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KNN, k=</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">algoritm</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="knnK">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="eigen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Eigenfaces, k=</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">algoritm</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="eigenK">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>120</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="eigenRC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Eigenfaces cu RC</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">algoritm</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="lanczos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lanczos, k=</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">algoritm</string>
    </attribute>
   </widget>
   <widget class="QComboBox" name="lanczosK">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="manhattan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Manhattan</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">normag</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="euclidiana">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Euclidiana</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">normag</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="infinit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Infinit</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">normag</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="cosinus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cosinus</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">normag</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="cauta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cauta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Database Configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algoritm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Norma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selecteazaPoza">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecteaza poza</string>
    </property>
   </widget>
   <widget class="QFrame" name="mesaj">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="statistici">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistici</string>
    </property>
   </widget>
   <widget class="QLabel" name="pozaCautata">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pixel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pozaGasita">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pixel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="algoritm"/>
  <buttongroup name="configureDB"/>
  <buttongroup name="chooseDB"/>
  <buttongroup name="normag"/>
 </buttongroups>
</ui>
